lib/order.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'menu'
+require_relative 'messager'
+
+class Order
+
+  attr_reader :list, :menu, :order, :total_cost, :message, :messager
+
+  def initialize(menu = Menu.new, messager = Messager.new)
+    @list = []
+    @menu = menu
+    @pizzas = menu.pizzas
+    @messager = messager
+  end
+
+  def read_menu
+    @menu.display_menu
+  end
+
+  def choose(item, n = 1)
+   n.times do @choice = @pizzas.select { |key, value| item.include? key }
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.